cumulusci/tasks/robotframework/template.html

```diff
@@ -0,0 +1,66 @@
+<html>
+  <head>
+    <script
+      src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+      integrity="sha256-3edrmyuQ0w65f8gfBsqowzjJe2iM6n0nKciPUp8y+7E="
+      crossorigin="anonymous">
+    </script>
+    <script>
+      function search() {
+          var search_string = $('#search').val().toLowerCase();
+          $('.kwtable .kwname').each(function() {
+              if ($(this).text().toLowerCase().indexOf(search_string) !== -1) {
+                  $(this).parent().show();
+              } else {
+                  $(this).parent().hide();
+              }
+          })
+      }
+      $(document).ready(function() {
+          $('#search').on('input', function() {
+              search()
+          })
+          $('#search').keyup(function() {
+              search()
+          })
+      });
+    </script>
+    <style>
+      {{stylesheet}}
+    </style>
+  </head>
+  <body>
+    <div class='container'>
```

Comment:
Not a big deal, but I'm distracted by the single-quoted HTML attributes, especially mixed with double quotes in the same file.

Reply:
That's a character weakness showing through. I've been terribly inconsistent about that my whole life and can't seem to shake it. I think I have a small tumor right next to the "quote" brain cell that sometimes swells and sometimes shrinks.

Reply:
We can probably solve it by setting up an autoformatter.

Reply:
fixed